Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -160,7 +168,7 @@ const StatusBar = React.createClass({
      * When translucent is set to true, the app will draw under the status bar.
      * This is useful when using a semi transparent status bar color.
      *
-     * @platform android
+     * @platform android windows
```

Comment:
Should I add this or not? I wonder what is the correct syntax 
`@platform android windows` or `@platform android, windows `